go.mod

```diff
@@ -1,39 +1,45 @@
 module github.com/Digital-Insight-Technologies-Ltd/gin-jwt-cognito
 
-go 1.20
+go 1.21.0
+
+toolchain go1.23.2
 
 require (
-	github.com/gin-gonic/gin v1.9.1
+	github.com/gin-gonic/gin v1.10.0
+	github.com/go-errors/errors v1.5.1
```

Comment:
Review and remove any unnecessary dependencies added in this PR to minimize the attack surface and reduce build times. Ensure that all added dependencies are essential for the middleware's functionality. [medium]